src/Apps/Artist/Routes/Overview/ArtistOverviewRoute.tsx

```diff
@@ -55,12 +35,6 @@ const ArtistRelatedArtistsRailQueryRenderer = loadable(
       component.ArtistRelatedArtistsRailQueryRenderer,
   }
 )
-const ArtistSellWithArtsyQueryRenderer = loadable(
-  () => import("./Components/ArtistSellWithArtsy"),
-  {
-    resolveComponent: component => component.ArtistSellWithArtsyQueryRenderer,
-  }
-)
 
 interface ArtistOverviewRouteProps {
   artist: any
```

Comment:
I know this isn't your doing but this is surprising to see. Should fix this before continuing.

Reply:
Yea, I was a little confused as to why these components were being passed in this way. I'm assuming using a loadable() might help with dynamically importing the components, but is an unnecessary over complication that could lead to performance issues right??

Reply:
A more straightforward approach would be to import the QueryRenderers and render them directly.

Reply:
Also, why would we need to lazy load here in the /about tab? Not sure...

Reply:
So yeah: you can remove the fragment entirely if we're not server rendering anything in this tab. The header is part of the route parent and will be the above the fold content that gets server rendered.

And yeah, I don't necessarily think it's needed to lazy load the components here; none of them should be particularly huge, but basically it's an optimization to ensure that we split the bundle on these tabbed routes.